src/closure.js

```diff
@@ -5,27 +5,61 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  return () => {
+    return ++count;
+  };
 };
 
 const counterFactory = () => {
   // Return an object that has two methods called `increment` and `decrement`.
   // `increment` should increment a counter variable in closure scope and return it.
   // `decrement` should decrement the counter variable and return it.
+  let count = 0;
+  return {
+    increment: () => { return ++count; },
```

Comment:
Array functions perform an implicit return if you don't use curly braces. So this can be shorthanded to `increment: () => ++count;` and `decrement: () => --count;`.